onnxruntime/contrib_ops/cpu/quantize_ops.cc

```diff
@@ -34,5 +34,14 @@ ONNX_CPU_OPERATOR_TYPED_MS_KERNEL(
         .TypeConstraint("T2", DataTypeImpl::GetTensorType<uint8_t>()),
     QuantizeLinear<uint8_t>);
 
+ONNX_CPU_OPERATOR_TYPED_MS_KERNEL(
+    QuantizeLinear,
+    1,
+    int8_t,
+    KernelDefBuilder()
```

Comment:
Now that this version exists, the bug about -127 to 127 range should also be fixed in the non-MLAS path for QuantizeLinear in the CPU EP.